packages/adminrouter/extra/src/test-harness/tests/conftest.py

```diff
@@ -33,7 +33,7 @@ def repo_is_ee():
 
 
 @pytest.fixture(scope='session')
-def mocker_s(repo_is_ee, syslog_mock, extra_lo_ips):
+def mocker_s(repo_is_ee, syslog_mock, extra_lo_ips, dns_server_mock_s):
```

Comment:
Comment that you need `dns_server_mock_s` explicitly because else the order of fixture loading matters.

❓ If we used a linter like `pylint` this would warn that `dns_server_mock_s` is an unused parameter. See https://docs.pytest.org/en/latest/fixture.html#using-fixtures-from-classes-modules-or-projects for a way to say "this is needed but not accessed in the test".